### PR TITLE
Create reaction roles

### LINKED ISSUE #6: Reaction roles
Bot should have reaction roles,

Configuration options:
* channel
* category, new category triggers a new message
* Reaction role:
  * role (e.g. @13-15)
  * description (e.g. 13-15 year old)
  * emoji
  * category (e.g. age)

### PR DESCRIPTION
**Pull request by mkevenaar**
Reaction roles feature 

## Description

TBD

## Related Issue

Fixes #6 

## Motivation and Context

New functionality

## How Has This Been Tested?

In the develop guild 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/bot.js [hunk 1]
```diff
@@ -85,6 +92,19 @@ export async function initBot() {
...
+	mongoose
+		.connect(MONGODB, {
+			useNewUrlParser: true,
+			useUnifiedTopology: true,
+		})
+		.then((_) => {
+			console.log('Connected to MongoDB');
+		})
+		.catch((err) => {
+			console.log('Unable to connect to MongoDB Database.\nError: ' + err);
```
Comment (Lucxjo) [inline src/bot.js:105]: Would it be better to use `console.error` here instead of `console.log`?

**Review by Lucxjo — COMMENTED** [review]
I know that a review wasn't particularly requested. Looks good so far but could you clear something up please

File: src/bot.js (hunk 1 above)
Comment (Lucxjo) [reply src/bot.js:105]: > Would it be better to use `console.error` here instead of `console.log`?

This is in reference to line 105, not 102

Comment (davidzwa) [reply src/bot.js:105]: Or consider using `winston`!

Comment (mkevenaar) [reply src/bot.js:107]: This is a discussion on it self

Comment (mkevenaar) [reply src/bot.js:107]: @davidzwa Do you want to add an issue about this and what we talked about over at the Nick bot?

**Review by Lucxjo — APPROVED** [review]
All looks good!

**Pull request merged**